src/commons/error.rs

```diff
@@ -423,7 +440,16 @@ impl fmt::Display for Error {
             Error::CaAuthorizationDuplicate(_ca, roa) => write!(f, "ROA '{}' already present", roa),
             Error::CaAuthorizationInvalidMaxLength(_ca, roa) => write!(f, "Invalid max length in ROA: '{}'", roa),
             Error::CaAuthorizationNotEntitled(_ca, roa) => write!(f, "Prefix in ROA '{}' not held by you", roa),
-            Error::RoaDeltaError(e) => write!(f, "ROA delta rejected:\n\n'{}' ", e),
+            Error::RoaDeltaError(_ca, e) => write!(f, "ROA delta rejected:\n\n'{}' ", e),
+
+            //-----------------------------------------------------------------
+            // Autonomous System Provider Authorization - ASPAs
+            //-----------------------------------------------------------------
+            Error::AspaCustomerAsNotEntitled(_ca, asn) => write!(f, "Customer ASN '{}' is not held by you", asn),
```

Comment:
Some of the message content above used AS rather than ASN. Should these be made consistent?